docs/MO_DG/prepare_model/FP16_Compression.md

```diff
@@ -3,15 +3,12 @@
 Model Optimizer can convert all floating-point weights to `FP16` data type. The resulting IR is called
 compressed `FP16` model.
 
-To compress the model, use the `--data_type` option:
+To compress the model, use the `--compress_to_fp16` option:
 
 ```
- mo --input_model INPUT_MODEL --data_type FP16
+ mo --input_model INPUT_MODEL --compress_to_fp16
 ```
 
-> **NOTE**: Using `--data_type FP32` will give no result and will not force `FP32` 
-> precision in the model. If the model was `FP16`, it will have `FP16` precision in IR as well.
-
```

Comment:
Please think about an old user who applied`--data_type FP16` or even `--data_type FP32`. Where can such users find information about how old and new options are connected? Leave them a note like 'Starting from 2022.3 release, option --data_type is deprecated, instead use --compress_to_fp16 in the following way: ...'.

Reply:
Added the note.